axelor-supplychain/src/main/java/com/axelor/apps/supplychain/service/SaleOrderLineServiceSupplyChainImpl.java

```diff
@@ -211,4 +214,37 @@ public void updateDeliveryState(SaleOrderLine saleOrderLine) {
       saleOrderLine.setDeliveryState(SaleOrderLineRepository.DELIVERY_STATE_DELIVERED);
     }
   }
+
+  @Override
+  public SaleOrderLine createPackLine(PackLine packLine, SaleOrder saleOrder)
+      throws AxelorException {
+    SaleOrderLine subLine = new SaleOrderLine();
+    Product subProduct = packLine.getProduct();
+    subLine.setProduct(subProduct);
+    subLine.setProductName(subProduct.getName());
+    subLine.setPrice(subProduct.getSalePrice());
+    subLine.setUnit(this.getSaleUnit(subLine));
+    subLine.setQty(new BigDecimal(packLine.getQuantity()));
+    subLine.setCompanyCostPrice(this.getCompanyCostPrice(saleOrder, subLine));
+    subLine.setSaleSupplySelect(subProduct.getSaleSupplySelect());
```

Comment:
Here, I assume that only saleSupplySelect assignation is added, so better to call the super method and add only this assignation to avoid duplicate code.